obelix/idefix/services/google/gcalendar.go

```diff
@@ -0,0 +1,122 @@
+package google
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Shopify/sarama"
+	redisv8 "github.com/go-redis/redis/v8"
+	"google.golang.org/api/calendar/v3"
+	"idefix/operator"
+	"idefix/producer"
+	"idefix/redis"
+	"log"
+	"time"
+)
+
+const (
+	ACTIVE = iota
+	NO_ACTIVE
+)
+
+type GCalendar struct {
+	date string
+}
+
+func (gc *GCalendar) getEventsList(srv *calendar.Service) (*calendar.Events, error) {
+	t := time.Now().Format(time.RFC3339)
+	events, err := srv.Events.List("primary").ShowDeleted(false).SingleEvents(true).TimeMin(t).MaxResults(1).OrderBy("startTime").Do()
```

Comment:
Multiline will look better and enforce maintainability even if it takes more line

```go
	events, err := srv.Events.List("primary").
	    ShowDeleted(false).
	    SingleEvents(true).
	    TimeMin(t).
	    MaxResults(1).
	    OrderBy("startTime").
	    Do()
```

Reply:
586e112212f1acb511d326766e650c83d6f88b7b